src/TcUnit.TestAdapter/Models/FunctionBlock_POU.cs

```diff
@@ -50,6 +50,9 @@ public static FunctionBlock_POU ParseFromFilePath(string filepath)
 
             var declaration = xFunctionBlock.Element("Declaration").Value;
 
+            functionBlock.Declaration = declaration;
+            functionBlock.Implementation = xFunctionBlock.Element("Implementation").Element("ST").Value;
```

Comment:
This might not be very efficient for large projects since it may load a lot of code into memory. We could look an alternative option that parses the file on the fly.